x-pack/solutions/observability/plugins/uptime/common/rules/alert_actions.test.ts

```diff
@@ -207,25 +204,75 @@ describe('Legacy Alert Actions factory', () => {
         defaultRecoverySubjectMessage: MonitorStatusTranslations.defaultRecoverySubjectMessage,
       },
     });
+
     expect(resp).toEqual([
       {
         group: 'recovered',
         id: 'f2a3b195-ed76-499a-805d-82d24d4eeba9',
         params: {
-          dedupKey: expect.any(String),
-          eventAction: 'resolve',
-          summary:
+          message:
             'Alert for monitor {{context.monitorName}} with url {{{context.monitorUrl}}} from {{context.observerLocation}} has recovered',
         },
       },
       {
         group: 'xpack.uptime.alerts.actionGroups.monitorStatus',
         id: 'f2a3b195-ed76-499a-805d-82d24d4eeba9',
         params: {
-          dedupKey: expect.any(String),
-          eventAction: 'trigger',
-          severity: 'error',
-          summary: MonitorStatusTranslations.defaultActionMessage,
+          message: MonitorStatusTranslations.defaultActionMessage,
+        },
+      },
+    ]);
+  });
+
+  it('generate expected action for slack webapi webhook action connector', async () => {
+    const resp = populateAlertActions({
+      groupId: MONITOR_STATUS.id,
+      defaultActions: [
+        {
+          actionTypeId: '.slack_api',
+          group: 'xpack.uptime.alerts.actionGroups.monitorStatus',
+          config: {
+            allowedChannels: [
+              { name: '#channel-name' },
+              { id: 'channel-id' },
```

Comment:
Just a nit. This is not possible, right? Only `id` is optional; `name` is mandatory.